Code/Source/Camera/ROS2CameraSensorComponent.h

```diff
@@ -45,6 +46,7 @@ namespace ROS2
         void FrequencyTick() override;
 
         std::shared_ptr<rclcpp::Publisher<sensor_msgs::msg::Image>> m_imagePublisher;
+        std::shared_ptr<rclcpp::Publisher<sensor_msgs::msg::CameraInfo>> m_cameraInfoPublisher;
```

Comment:
Could it be an unique_ptr instead? (Same above).

Reply:
Since it is a publisher it is created by rclcpp node API
```
        auto ros2Node = ROS2Interface::Get()->GetNode();
        ...
        m_imagePublisher =
            ros2Node->create_publisher<sensor_msgs::msg::Image>(cameraImageFullTopic.data(), cameraImagePublisherConfig.GetQoS());
```
And it returns `shared_ptr` and there is no straightforward way to make `unique_ptr` from it